src/nxt_h1proto.c

```diff
@@ -518,6 +522,10 @@ nxt_h1p_conn_request_init(nxt_task_t *task, void *obj, void *data)
             h1p->parser.discard_unsafe_fields = skcf->discard_unsafe_fields;
 
             nxt_h1p_conn_request_header_parse(task, c, h1p);
+#if (NXT_HAVE_OTEL)
```

Comment:
`NXT_HAVE_OTEL` means Unit support otel feature,
 I wonder what's the requirement of when to trace http request?
With this implementation, it means all the requests will be traced if otel is compiled into Unit.
Is it our intention?

Reply:
Yeah, I think we want to *think* about that, do we want it enabled at compile time, runtime, reconfigure...

Do we want to be able to enable/disable per application/route...